src/tests/unit/components/Articles.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Articles from 'components/Articles';
+import Article from 'components/Article';
+import TagList from 'components/TagList';
+
+const firstArticle = { title: 'Authors Haven introduction', slug: 'title-1', tagList: [] };
+
+const articles = [
+  firstArticle,
+  { title: 'Authors Haven introduction', slug: 'title-2', tagList: [] },
+];
+
+describe('Articles', () => {
+  test('It Articles  articles all Article', () => {
+    const wrapper = shallow(<Articles articles={articles} />);
+    expect(wrapper.find('Article')).toHaveLength(2);
+  });
+});
+
+describe('Article', () => {
+  test('It Article shows the right title', () => {
+    const wrapper = shallow(<Article article={firstArticle} />);
+    expect(wrapper.find('.card-title').text()).toEqual(firstArticle.title);
```

Comment:
leave line between line 23 and 24. Kindly change for the rest of the file as well